src/autopas/options/TraversalOption.h

```diff
@@ -22,40 +22,126 @@ class TraversalOption : public Option<TraversalOption> {
    */
   enum Value {
     // DirectSum Traversals:
+    /**
+     * DSSequentialTraversal : Sequential double loop over all particles.
+     */
     ds_sequential,
 
     // LinkedCell Traversals:
+    /**
+     * LCSlicedTraversal : 1D equidistant slicing of the domain with one slice per thread. One lock per slice interface.
+     * Uses c08 base-step per cell. Minimal scheduling overhead at the cost of no load balancing at all.
+     */
     lc_sliced,
+    /**
+     * LCSlicedBalancedTraversal : Same as lc_sliced but tries to balance slice thickness according to a given
+     * LoadEstimatorOption.
+     */
     lc_sliced_balanced,
+    /**
+     * LCSlicedC02Traversal : 1D slicing with as many slices of minimal thickness as possible. No locks but two way
+     * coloring of slices.
+     */
     lc_sliced_c02,
+    /**
+     * LCC01Traversal : Every cell is interacted with all neighbors. Is not compatible with Newton3 thus embarrassingly
+     * parallel. Good load balancing and no overhead.
+     */
     lc_c01,
+    /**
+     * LCC01Traversal : Same as LCC01Traversal but SoAs are combined into a circular buffer and the domain is traversed
+     * line wise.
+     */
     lc_c01_combined_SoA,
+    /**
+     * LCC01CudaTraversal : CUDA version of LCC01Traversal.
+     */
     lc_c01_cuda,
+    /**
+     * LCC04Traversal : Four way domain coloring using plus shaped clusters of cells that are processed with the c08
+     * base-step. Less scheduling overhead than LCC08Traversal because of fewer barriers but more coarse grained.
```

Comment:
```suggestion
     * LCC04Traversal : Four-way domain coloring using plus-shaped clusters of cells that are processed with the c08
     * base-step. Less scheduling overhead than LCC08Traversal because of fewer barriers but more coarse-grained.
```